core/biogeophys/cable_diag.F90

```diff
@@ -35,7 +35,7 @@
 
 !#define UM_BUILD YES
 MODULE cable_diag_module
-  use cable_def_types_mod, only : r_2
+  use cable_def_types_mod, only : r2
```

Comment:
Unused at this level, could remove since it's imported within individual subroutines.